### DISCUSSION TITLE
Does akvorado support jflow v5?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1410] [Q&A]
Hi all,

My current setup juniper is using routing-instance "virtual-router". Based on this url https://supportportal.juniper.net/s/article/MX-Example-Configuring-forwarding-options-to-send-IPFIX-to-remote-flow-server-inside-routing-instance?language=en_US it mention flow export just can do on version5. May i know the akvorado support jflow v5? Thanks and appreciate any feedback.

Thanks

**Comment by vincentbernat** [comment]
Yes, this is supported.

**Reply by kronicklez** [reply]
Hi vincent,

Do i need to change anything in inlet.yaml or akvorado.yaml if i'm use jflowv5? Or in akvorado dont need change anything same as using ipfix and jflowv9?

Thanks

**Reply by vincentbernat** [reply]
No change needed. The version is encoded in the first byte and autodetected.
